documentation/migrating/01-migrating.md

```diff
@@ -4,4 +4,4 @@ title: Migrating from Sapper
 
 SvelteKit is the successor to Sapper and shares many elements of its design.
 
-If you have an existing Sapper app that you plan to migrate to SvelteKit, there are a number of changes you will need to make. You may find it helpful to view the [examples](https://github.com/sveltejs/kit/tree/master/examples) while migrating.
+If you have an existing Sapper app that you plan to migrate to SvelteKit, there are a number of changes you will need to make. You may find it helpful to view some examples from our [sites](https://github.com/sveltejs/kit/tree/master/examples) repository while migrating.
```

Comment:
you didn't update the link :grin:  but there's a few more examples besides the ones in the sites repo
```suggestion
If you have an existing Sapper app that you plan to migrate to SvelteKit, there are a number of changes you will need to make. You may find it helpful to view [some examples](https://kit.svelte.dev/docs#additional-resources-examples) while migrating.
```

Reply:
oops! not sure how I missed that one 😅